lib/HALSim/src/Board.cpp

```diff
@@ -68,6 +74,17 @@ bool Board::init()
 {
     bool isReady = true;
 
+#if _WIN32
+    WORD wVersionRequested = MAKEWORD(2,2);
+    WSADATA wsaData;
+    int result = WSAStartup(wVersionRequested, &wsaData);
+
+    if (0 != result)
+    {
+        LOG_ERROR("WSAStartup error %d", result);
```

Comment:
Shouldn't the return value for init() be influenced?

Reply:
will now caus init() to fail.